src/main/java/graphql/Scalars.java

```diff
@@ -286,7 +280,7 @@ public Double serialize(Object input) {
         public Double parseValue(Object input) {
```

Comment:
There were some copy and pasta errors in the messages.  I fixed them and put the offending object in 'quotes'